xbmc/addons/AddonManager.h

```diff
@@ -240,9 +246,9 @@ namespace ADDON
     /*! \brief Stop all services addons.
     */
     void StopServices(const bool onlylogin);
-    static AddonPtr Factory(const cp_plugin_info_t* plugin, TYPE type);
-    static AddonPtr Factory(const cp_plugin_info_t* plugin, TYPE type, CAddonBuilder& builder);
-    static void FillCpluffMetadata(const cp_plugin_info_t* plugin, CAddonBuilder& builder);
+    AddonPtr Factory(const cp_plugin_info_t* plugin, TYPE type);
+    AddonPtr Factory(const cp_plugin_info_t* plugin, TYPE type, CAddonBuilder& builder);
+    void FillCpluffMetadata(const cp_plugin_info_t* plugin, CAddonBuilder& builder);
```

Comment:
Looks unrelated to this commit. 

it's static for a good reason: the keep it pure. it should not have access to anything internally, only the cpluff helper functions.